base_revision/tests/test_base_revision.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2013 Agile Business Group sagl (<http://www.agilebg.com>)
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2018 Dreambits Technologies Pvt. Ltd. (<http://dreambits.in>)
+# Copyright 2020 Ecosoft (<http://ecosoft.co.th>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+
+from .base_revision_tester import BaseRevisionTester
+from .common import setup_test_model, teardown_test_model
+
+
+class TestBaseRevision(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestBaseRevision, cls).setUpClass()
```

Comment:
Use the fakeModelLoader: https://github.com/OCA/edi/blob/13.0/edi/tests/test_consumer_mixin.py#L24

Reply:
TIL

Reply:
Fixed. Thanks for your note about this.